README.md

```diff
@@ -1 +1,182 @@
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [💾 Database](#-database)
+    - [Usage](#usage)
+    - [Run tests](#run-tests)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Book a Hotel <a name="about-project"></a>
+
+> The Book a Hotel 🏩, our API will provide functionality to create, read, and destroy data related to hotels. It will serve as the server-side component responsible for handling requests and managing the database. The React client will interact with this backend to perform actions such as creating new hotels and reservations, see the hotel details, and removing hotels from the system.
+
+## 🛠 Visit our deployed Link:<a>
+https://book-an-appointment-cmm0.onrender.com/api-docs/v1/swagger.yaml
+</a>
+
+## 🛠 Visit our Frontend link:<a>
+https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-frontend/pull/7
+</a>
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+    <ul>
+    <li><a href="https://react.dev/">React</a></li>
+    <li><a href="https://redux.js.org/">Redux</a></li>
+  </ul>
+</details>
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://rubyonrails.org/">Ruby on Rails</a></li>
+  </ul>
+  </details>
+  <details>
+  <summary>Database</summary>
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Data Model**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+
+> You need the following tools be installed in your computer:
+
+> - [Rails](https://guides.rubyonrails.org/)
+> - [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+> - [Ruby](https://github.com/microverseinc/curriculum-ruby/blob/main/simple-ruby/articles/ruby_installation_instructions.md)
+
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+cd my-prefered-folder
+  
+git clone https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend.git
+
+```
+
+### Install
+
+Install this project with:
+
+```sh
+cd finalGroupCapstone-backend
+  
+bundle
+```
+
+### 💾 Database
+
+Create the databases, run migrations, and populate it with seed data:
+
+```sh
+rails db:create
+rails db:migrate
+rails db:seed
+```
+
+### Usage
+
+To run the project run the following commands:
+
+```sh
+rails s
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+
+```sh
+rspec spec --exclude-pattern "spec/integration/api/*_spec.rb"
```

Comment:
- [ ] After setting up and testing the database, if I run the tests according to the instructions this happens:

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/95ee6d03-a48f-4949-aa8c-f15dad04f10e)

Kindly, fix your tests so they can be run and they all pass.  👍 

_note:  The database was set up before attempting to run the tests:_

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/868f3fb2-7d57-4866-966f-c78336b5e55c)

![image](https://github.com/IvonneBenitesRodriguez/finalGroupCapstone-backend/assets/57421823/be60f6d2-d0fc-4cc3-9c8b-9f62b54c530e)